addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -497,8 +497,18 @@ options.registry.layout_column = options.Class.extend({
 
         if (count > 0) {
             var $lastColumn = this.$target.children().last();
-            for (var i = 0 ; i < count ; i++) {
-                $lastColumn.clone().insertAfter($lastColumn);
+            for (var i = 0; i < count; i++) {
+                var $clone = $lastColumn.clone().insertAfter($lastColumn);
+                this.trigger_up('call_for_each_child_snippet', {
+                    $snippet: $clone,
+                    callback: function (editor, $snippet) {
+                        for (var i in editor.styles) {
+                            editor.styles[i].onClone({
+                                isCurrent: ($snippet.is($clone)),
+                            });
+                        }
+                    },
+                });
```

Comment:
Should not be the role of the option to do this kind of stuff, especially since it is purely duplicated code. Create a method in the editor to share that code and here trigger an event that will be caught in the editor.

Reply:
Looks much cleaner! Done :)